scripts/generate_test_code.py

```diff
@@ -651,6 +651,9 @@ def parse_function_code(funcs_f, dependencies, suite_dependencies):
         raise GeneratorInputError("file: %s - Test functions not found!" %
                                   funcs_f.name)
 
+    # Make the test function static
+    code = code.replace('void', 'static void', 1)
```

Comment:
> This could be better to modify in line 647 above.
> 
> ```
>         args, local_vars, args_dispatch = parse_function_arguments(
>             line)
>         code += line.replace('void', 'static void', 1)
>         break
> ```

This feels kind of mingled; I wanted to be consistent with these lines below:

https://github.com/Mbed-TLS/mbedtls-framework/blob/3d25f35d68528317ab58db19485727b0ed9a53cc/scripts/generate_test_code.py#L657-L659

https://github.com/Mbed-TLS/mbedtls-framework/blob/3d25f35d68528317ab58db19485727b0ed9a53cc/scripts/generate_test_code.py#L664

IMHO more readable and easier to maintain, without interfering with the matching logic above.

> 
> But assuming we want to only replace a `void ` in the begining of the file we could achieve the same with:
> 
> ```
> code += re.sub(r'^void .', 'static void ',line)
> ```

This breaks my local build again because not all functions are made static. Did you test it?

> 
> Either way works

Shall we keep it like before?

Reply:
Yes it feels trivial change to block the pr. I will look how to optimise the generate_test_code in the future.